src/backends.rs

```diff
@@ -299,6 +300,11 @@ impl GitHubBackend {
 }
 
 /// Get the currently configured araki backend.
-pub fn get_current_backend() -> Result<impl Backend, BackendError> {
-    GitHubBackend::new()
+pub fn get_current_backend(settings: Config) -> Result<impl Backend, BackendError> {
+    match settings.get_string("backend")?.to_lowercase().trim() {
```

Comment:
hmmm, I think this implementation leaves will leave us with some weird behaviour. Like, what if:
* a user has already setup their project with github
* they are doing other stuff in their terminal and have set their env var `ARAKI_BACKEND=gitlab`. Maybe the other stuff they are doing is setting up another project with a gitlab backend?
* then they cd into their project that is backed by github. Now if they want to auth for this project their backend will be incorrectly configured to be gitlab.

The `backend` is probably something that should be configured per project. And probably not have any per run, or global configuration. Probably a globally configured default is ok.